docs/support/migration.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Migration
+---
+
+# Migrating between SDK versions
+
+While the Packs team strives to maintain backwards compatibility between SDK versions, there may occasionally be times where you need to change your code to work with a new SDK version. This page will include instructions on how to perform the migration. A detailed list of all changes in the SDK can be found in the [Changelog][changelog].
+
+
+## Version 0.11.0
+
+In preparation for a public launch, we intentionally introduced a number of breaking changes to make the SDK easier to use and understand. We hope to avoid breaking changes on this scale in the future.
+
+
+### Rename schema fields `id`, `primary`, and `featured`
+
+__Affects__: Packs that define a [schema][schemas].<br>
+__Action Required__: Rename only.
+
+To better reflect their meaning, we've renamed certain fields in the schema definition. Specifically:
+
+- `id` --> `idProperty`
+- `primary` --> `displayProperty`
+- `featured` --> `featuredProperties`
+
+Only the names of these fields have changed, and it there should be no impact on the functionality of your Pack.
+
+```{.ts hl_lines="8-10"}
+let MovieSchema = coda.makeObjectSchema({
+  properties: {
+    title: { type: coda.ValueType.String },
+    year: { type: coda.ValueType.Number },
+    movieId: { type: coda.ValueType.String },
+    // ...
+  },
+  idProperty: "movieId",
+  displayProperty: "title",
+  featuredProperties: ["year"],
+  // ...
+});
+```
+
+
+### Add `identityName` to dynamic sync tables
+
+__Affects__: Packs that include a dynamic sync table.<br>
+__Action Required__: Add new code.
+
+Like regular sync tables, dynamic sync tables now require the `identityName` field to be set. This will be used along with the dynamic URL to set the identity of the table. You no longer need to set the `identity` field of the schema generated in the `getSchema` function, as it will be constructed for you automatically.
+
+To ensure that existing dynamic sync tables in your user's docs continue to work we strongly recommend you set this value to be the same as the name of the table.
```

Comment:
Can we be more explicit? We could add "If you don't, references to your dynamic sync table may break and row updates from action return values may stop working."?